demos/vc_issuer/vc_demo_issuer.did

```diff
@@ -102,6 +113,7 @@ service: (opt IssuerConfig) -> {
     vc_consent_message : (Icrc21VcConsentMessageRequest) -> (variant { Ok : Icrc21ConsentInfo; Err : Icrc21Error;});
     prepare_credential : (PrepareCredentialRequest) -> (variant { Ok : PreparedCredentialData; Err : IssueCredentialError;});
     get_credential : (GetCredentialRequest) -> (variant { Ok : IssuedCredentialData; Err : IssueCredentialError;}) query;
+    derivation_origin : (text) -> (variant {Ok: DerivationOriginData; Err: DerivationOriginError});
```

Comment:
What do you think, making the method also take a `record`? This would allow to name the `text` parameter and would make the method future proof to the addition of further arguments.

Reply:
👍 or additionally explain what the parameter is/does

Reply:
Actually, what is the parameter for?

Reply:
A single issuer-BE could be serving several frontends.

Reply:
Good idea, changed to a record.